packages/gatsby-source-contentful/src/__tests__/fetch.js

```diff
@@ -1,3 +1,11 @@
+/**
+ * @jest-environment node
+ */
+
+import nock from "nock"
+
+nock.disableNetConnect()
+
```

Comment:
This nock mock can be removed as soon Contentful SDK supports the tags collection endpoint.